### ISSUE TITLE
Diverse Rarities

### ISSUE THREAD

**Issue by ChristopherTheBoss** [ExcessiveAmountsOfZombies/Croptopia#320]
### Make sure you are not opening a duplicate.

- [X] [I have searched the issue tracker](https://github.com/ExcessiveAmountsOfZombies/Croptopia/issues?q=) and did not find an issue describing my bug.

### Description of your suggestion.

I play Minecraft vegetarian. Something that I first viewed as a punishment for choosing not kill animals for food is how long it would take to get a stable food supply in the beginning of the game.  However, as I played the game more, I discovered that, facing the effectively infinite procedurally generated world, the scarcity of harvestable foods provided a fun challenge.

I love Croptopia and most of the foods that come with it. One immediate change in gameplay this mod provides is how abundant plant foods are from the world's inception.  I am overwhelmed with things to eat, there is such a variety that I feel like everywhere in the overworld is the produce section of a grocery store. In the base game there is always a sense of progression from farming wheat, to having all 6 different seeds planted and growing at your base.

Ideally Croptopia should act as an expansion of existing crops, while also making the opening of the game less harsh. I would love to have to spend multiple (Minecraft) days exploring to find certain exotic fruits 🍑, while others 🍎 are readily spotted. Something about searching 2 thousand blocks from my base to find mangoes, then finding a grove of them. There would then be a choice of taking a few trees with all the way back home or setting up a base there and live as a Mad Lord of Mango.

TL;DR: My suggestion/feature request is for Croptopia to have sliders for the frequency of each naturally spawning plant. There could also be a slider for overall abundance that scales each slider in unison. I hope that this is possible, thanks for your consideration.
